content/en/real_user_monitoring/guide/mobile-sdk-upgrade.md

```diff
@@ -720,6 +762,102 @@ For instructions on setting up Mobile Session Replay, see [Mobile Session Replay
 
 {{% /tab %}}
+
+{{% tab "Flutter" %}}
+
+## SDK Configuration Changes
+
+Certain configuration properties have been moved or renamed to support modularity in Datadog's native SDKs.
+
+The following structures have been renamed:
+
+| `1.x` | `2.x` |
+| `DdSdkConfiguration` | `DatadogConfiguration` |
+| `LoggingConfiguartion` | `DatadogLoggingConfiguration` |
+| `RumConfiguration` | `DatadogRumConfiguration` |
+| `DdSdkExistingConfiguration` | `DatadogAttachConfiguration` |
+
+The following properties have changed:
+
+| `1.x` | `2.x` | Notes |
+|-------|-------|-------|
+| `DdSdkConfiguration.trackingConsent`| Removed | Part of `Datadog.initialize` | |
+| `DdSdkConfiguration.customEndpoint` | Removed | Now configured per-feature | |
+| `DdSdkConfiguration.serviceName` | `DatadogConfiguration.service` | |
+| `DdSdkConfiguration.logEventMapper` | `DatadogLoggingConfiguration.eventMapper` | |
+| `DdSdkConfiguration.customLogsEndpoint` | `DatadogLoggingConfiguration.customEndpoint` | |
+| `DdSdkConfiguration.telemetrySampleRate` | `DatadogRumConfiguration.telemetrySampleRate` | |
+
+In addition, the following APIs have changed:
+
+| `1.x` | `2.x` | Notes |
+|-------|-------|-------|
+| `Verbosity` | Removed | See `CoreLoggerLevel` or `LogLevel` |
+| `DdLogs DatadogSdk.logs` | `DatadogLogging DatadogSdk.logs` | Type changed |
+| `DdRum DatadogSdk.rum` | `DatadogRum DatadogSdk.rum` | Type changed
+| `Verbosity DatadogSdk.sdkVerbosity` | `CoreLoggerLevel DatadogSdk.sdkVerbosity` |
+| `DatadogSdk.runApp` | `DatadogSdk.runApp` | Added `trackingConsent` parameter |
+| `DatadogSdk.initialize` | `DatadogSdk.initialize` | Added `trackingConsent` parameter |
+| `DatadogSdk.createLogger` | `DatadogLogging.createLogger` | Moved |
+
+## Flutter Web Changes
+
+Clients using Flutter Web should update to using the Datadog Browser SDK v5.  Change the following import in your `index.html`:
+
+```diff
+-  <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/datadog-logs-v4.js"></script>
+-  <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/datadog-rum-slim-v4.js"></script>
++  <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/us1/v5/datadog-logs.js"></script>
++  <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/us1/v5/datadog-rum-slim.js"></script>
+```
+
+Note that Datadog provides one CDN bundle per site. See the [Browser SDK README](https://github.com/DataDog/browser-sdk/#cdn-bundles) for a list of all site URLs.
+
+## Logs Product Changes
+
+As with `1.x`, Datadog Logging can be enabled by setting the `DatadogConfiguration.loggingConfiguration` member. However, unlike `1.x`, Datadog will not create a default logger for you. `DatadogSdk.logs` is now and instance of `DatadogLogging`, which can be used to create logs. Many options were moved to `DatadogLoggerConfiguration` to give developers more granular support over individual loggers.
```

Comment:
```suggestion
As with v1, Datadog Logging can be enabled by setting the `DatadogConfiguration.loggingConfiguration` member. However, unlike v1, Datadog does not create a default logger for you. `DatadogSdk.logs` is now an instance of `DatadogLogging`, which can be used to create logs. Many options were moved to `DatadogLoggerConfiguration` to give developers more granular support over individual loggers.
```
We use "v1" and "v2" above in the tabs, so let's be consistent with that (outside of the tables, where I think it makes sense to use 1.x etc.